lib/email_tracking_retriever.py

```diff
@@ -285,10 +286,21 @@ def clean_email_content(email_str) -> str:
   return email_str
 
 
-def log_in_if_necessary(driver: WebDriver):
+def log_in_if_necessary(driver: WebDriver, config):
   driver.get('https://www.amazon.com/gp/your-account/order-history/ref=ppx_yo_dt_b_orders')
   orders_containers = driver.find_elements_by_id('ordersContainer')
   if len(orders_containers) == 0:
     # likely a login screen
-    driver.find_element_by_css_selector('form[name="signIn"]')
-    input('Please log in to an Amazon account on the opened Chrome profile. Hit ENTER when done.')
+    if 'amazon' in config:
+      print("Signing into Amazon ...")
+      driver.find_element_by_css_selector('form[name="signIn"]')
```

Comment:
i do not believe we need this line

Reply:
Only left it since it was there before, will remove